apps/studio/src/components/CmsSidebar/CmsSideNav.tsx

```diff
@@ -118,86 +166,41 @@ const SideNavItem = ({
         {/* so that we don't issue multiple db reads on load */}
         {({ isExpanded }) => {
           return (
-            <>
+            <Box pos="relative">
               {/* NOTE: required for focus ring */}
-              <Box m="4px">
-                <AccordionButton
-                  boxSizing="border-box"
-                  disabled={isDisabled}
-                  borderRadius="0.25rem"
-                  _focusVisible={{
-                    boxShadow: "none !important",
-                    outline: `2px solid var(--chakra-colors-utility-focus-default)`,
-                    outlineOffset: "0.125rem",
-                    _dark: {
-                      outline: `2px solid var(--chakra-colors-utility-focus-inverse)`,
-                    },
-                  }}
-                  _hover={{
-                    backgroundColor: "interaction.muted.main.hover",
-                  }}
-                  _active={{
-                    backgroundColor: "interaction.muted.main.active",
-                    textColor: "interaction.main.default",
-                  }}
-                  onDoubleClick={async () => {
-                    const urlType = getResourceType(resourceType)
-                    if (resourceType === "RootPage") {
-                      return router.push({
-                        pathname: "/sites/[siteId]",
-                        query: {
-                          siteId,
-                        },
-                      })
-                    }
-                    return router.push({
-                      pathname: "/sites/[siteId]/[resourceType]/[id]",
-                      query: {
-                        siteId,
-                        resourceType: urlType,
-                        id: resourceId,
-                      },
-                    })
-                  }}
+              <SideNavRow
+                leftIcon={<Box w="1rem" />}
+                isActive={isCurResourceActive}
+                href={
+                  resourceType === ResourceType.RootPage
+                    ? `/sites/${siteId}`
+                    : `/sites/${siteId}/${urlType}/${resourceId}`
+                }
+              >
+                <Icon fill="base.content.default" as={icon} flexShrink={0} />
+                <Text
+                  ml="0.25rem"
+                  noOfLines={1}
+                  textColor="base.content.default"
+                  textAlign="left"
+                  textStyle="subhead-2"
                 >
-                  <Flex
-                    w="full"
-                    color="base.content.default"
-                    alignItems="center"
+                  {permalink}
+                </Text>
+              </SideNavRow>
+              {isAllowedToHaveChildren(resourceType) &&
+                (isLoading || hasChildren) && (
+                  <AccordionButton
+                    disabled={isLoading}
```

Comment:
Nit: use the isDisabled prop instead for consistency? (ignore if it doesn’t exist)

Reply:
sadly doesn't exist
<img width="1434" alt="image" src="https://github.com/user-attachments/assets/17983c3f-b08e-4938-9af4-e3734c375265">